### ISSUE TITLE
3-3,高阶API示范 DNN模型的示例跟中阶API一样

### ISSUE THREAD

**Issue by qiu-nian** [lyhue1991/eat_tensorflow2_in_30_days#63]
按照高阶API的线性回归模型中的示范，建模过程应该是：
（1）通过 models.Sequential()的方式构建模型
（2）add()添加网络层
（3）定义loss， metric， optimizer
（4）通过compile()的方式配置模型训练中的各种参数
（5）model.fit()的方式训练模型

但是在高阶API示例的DNN模型的建模中，使用的是build()的方式，整体框架基本跟中阶API示例中DNN模型的建模过程一模一样，感觉这个是不是给错了示例了。
